### DISCUSSION TITLE
How do I make the HUD/On Scree Menus brigher, change their color?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [Stellarium/stellarium#3031] [Q&A] [answered]
They are black/dark and almost impossible to read when the location is under night.  Even when the location is in daylight, they're hard to read.  Is there any way to change their color or the gamma?  Something like light green would be much easier to read. 

Obviously, each person and each scenario is different so this should be something you can easier adjust with some hotkeys or something.

I see that the User Guide says you edit the CSS sheet for the app to change various colors.  That's ridiculously more complicated than it should be.  It shouldn't even require raising a menu - it should be a hotkey.  Something like Novalogic F22 flight simulator has hotkeys to change the color of the aircraft HUD so it's visible against any background.  Stellarium should have something similar.  By HUD in this case, I'm referring to Stellariums on-screen icons/menus on the bottom and left of the screen.  They are basically invisible out-of-the box in daylight locations.

**Comment by gzotti** [comment, marked as answer]
User Guide 6.2.3 describes ways to change GUI colors. This is a new feature. If you need more, feel free to make it. This is how free and open-source programs are created and developed. How many developers created the "Novalogic F22 flight simulator" and could not go flying themselves instead? Were they compensated for this (salary)? The few core developers here are not obliged to do everything that a program "should have" in your opinion. A moderately dark environment is always recommended. 

https://github.com/Stellarium/stellarium/wiki/FAQ#why-dont-you-implement
